src/components/PhotoDetails.vue

```diff
@@ -13,26 +13,26 @@
     <div class="column image-info">
       <section class="tabs" >
         <ul>
-          <li :class="tabClass(0, 'a')">
-            <a href="#panel0" :aria-selected="activeTab == 0" @click.prevent="setActiveTab(0)">
-              Attribution
-            </a>
-          </li>
           <li :class="tabClass(1, 'a')">
             <a href="#panel1" :aria-selected="activeTab == 1" @click.prevent="setActiveTab(1)">
               Info
             </a>
           </li>
-          <li :class="tabClass(2, 'a')" v-if="watermarkEnabled">
-            <a href="#panel2" :aria-selected="activeTab == 2" @click.prevent="setActiveTab(2)">
-              Download
+          <li :class="tabClass(0, 'a')">
```

Comment:
You should change the order of the markup elements, not the order of the indices

Reply:
@brenoferreira I believe I made the necessary changes please do review.